src/components/Widget/index.tsx

```diff
@@ -49,3 +49,9 @@ export default function Widget({ defaultToken, onReviewSwapClick }: WidgetProps)
     </>
   )
 }
+
+export function WidgetSkeleton() {
+  const darkMode = useIsDarkMode()
+  const theme = useMemo(() => (darkMode ? DARK_THEME : LIGHT_THEME), [darkMode])
+  return <SwapWidgetSkeleton theme={theme} width={WIDGET_WIDTH} />
```

Comment:
`DARK_THEME` and `LIGHT_THEME` are referentially stable, so you do not need to memoize `theme`.

    return <SwapWidgetSkeleton theme={darkMode ? DARK_THEME : LIGHT_THEME} widget={WIDGET_WIDTH} />

Reply:
I was copying over what was in <Widget/>, I'll remove from both